pkg/browser/open.go

```diff
@@ -0,0 +1,45 @@
+package browser
+
+import (
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+// OpenURL opens the specified URL in the default browser of the user.
+// Snippet source: https://gist.github.com/sevkin/9798d67b2cb9d07cb05f89f14ba682f8?permalink_comment_id=5084817#gistcomment-5084817
+func OpenURL(url string) error {
+	var cmd string
+	var args []string
+
+	switch runtime.GOOS {
+	case "windows":
+		cmd = "cmd"
+		args = []string{"/c", "start"}
+	case "darwin":
+		cmd = "open"
+		args = []string{url}
+	default: // "linux", "freebsd", "openbsd", "netbsd"
+		if isWSL() {
+			cmd = "cmd.exe"
+			args = []string{"/c", "start", url}
+		} else {
+			cmd = "xdg-open"
+			args = []string{url}
+		}
+	}
+	if len(args) > 1 {
+		// args[0] is used for 'start' command argument, to prevent issues with URLs starting with a quote
+		args = append(args[:1], append([]string{""}, args[1:]...)...)
+	}
+	return exec.Command(cmd, args...).Start()
+}
+
+// isWSL checks if the Go program is running inside Windows Subsystem for Linux
+func isWSL() bool {
+	releaseData, err := exec.Command("uname", "-r").Output()
+	if err != nil {
```

Comment:
can we handle the error somehow? e.g. file not found?

Reply:
I'm not sure what to handle here. It expects that this command will work in WSL. If doesn't work, it's not WSL

Reply:
yeah, I would also support adding some context or logging.

Reply:
I replaced this code with the lib call